src/Drive.php

```diff
@@ -583,9 +587,13 @@ public function emptyTrashbin(): bool
      * @throws UnauthorizedException
      * @throws InvalidResponseException
      * @throws InternalServerErrorException
+     * @throws EndPointNotImplementedException
      */
     public function getRoles(): array
     {
+        if((version_compare($this->ocisVersion, '6.0.0', '<='))) {
```

Comment:
when was the endpoint implemented? 6.0.1?
maybe, in a comment, we should mention the exact version where it was implemented, to help future readers.